### DISCUSSION TITLE
LXQt System theme issue on Lubuntu

### DISCUSSION THREAD

**Discussion by edwardp3** [lxqt/lxqt#2098] [Q&A] [answered]
I installed both Lubuntu 21.10 and the PC Linux OS Community LXQt distro (PCLOS) 21.10 in virtual machines and they both run well.

I noticed an oddity with the System theme on Lubuntu. When an app is open, the icon (?) that appears on the LXQt panel, is not displaying the app icon (in the attached examples, Firefox) and the text that is taken from the window title, is displayed at about the mid-way point instead of at the beginning on the left. 

The same icons in PCLOS and Debian 11 (the host system) display the same correctly. I am attaching screengrabs showing this on Lubuntu and PCLOS.

I do not know if this is an issue with the System theme itself, or with Lubuntu.

The top row in the images below is the OS running in the virtual machine, the bottom row is Debian.

![lxqt-panel-lubuntu](https://user-images.githubusercontent.com/73129450/138369536-ba7d973d-addb-4574-8add-09e77b938826.png) (Lubuntu)

![lxqt-panel-pclos](https://user-images.githubusercontent.com/73129450/138369548-e27963cd-f4c4-419c-886e-762bab01cfbb.png) (PCLOS)

Thank you in advance for any insights/suggestions.

**Comment by tsujan** [comment, marked as answer]
If your LXQt version isn't too old, it's a bug in the Breeze widget style. See https://github.com/lxqt/lxqt/issues/1525#issuecomment-406907067

**Reply by edwardp3** [reply]
Hi Tsujan,

Thanks for replying. Both Lubuntu and PCLOS include LXQt 0.17.

This was a simple fix, I changed the widget style from Breeze to Fusion and that corrected the Lubuntu issue.

**Reply by tsujan** [reply]
Hi,

> Both Lubuntu and PCLOS include LXQt 0.17.

0.17 is good.

> I changed the widget style from Breeze to Fusion

Yes, any widget style other than Breeze and Oxygen should work fine.

I'm not saying that Breeze is "bad" — on the contrary, it's a good style. But it has bugs that KDE devs aren't willing to fix (I know of 3: this one, an issue in window dragging, and another problem whose description is beyond this comment). On the other hand, Breeze is superior to Fusion in some respects.
